src/components/AddItem.js

```diff
@@ -1,34 +1,78 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { withFirestore } from 'react-firestore';
+import '../styles/AddItem.css';
+import { v4 as uuidv4 } from 'uuid';
 
 const AddItem = props => {
+  const emptyShoppingItem = {
+    name: '',
+    lastPurchasedDate: null,
+    nextPurchase: 7,
+  };
+
   const { firestore } = props;
-  const [enteredValue, setEnteredValue] = useState();
+  const [enteredValue, setEnteredValue] = useState(emptyShoppingItem);
 
-  const addMsg = e => {
+  const handleChange = e => {
+    setEnteredValue({ ...enteredValue, [e.target.name]: e.target.value });
+  };
+
+  const addItem = e => {
```

Comment:
Just a thought: Not sure how important this is but since the component name is `AddItem`, maybe we can change the function name to something else here to avoid confusion?  (Maybe: addNewItem).

Reply:
Good thoughts here, @Lukabaramishvili! Not a blocker, but could be nice to name this even more explicitly and say that it's adding the item to Firebase? 🙌